### PR TITLE
Add dog resource

### PR DESCRIPTION
**Pull request by cat-does**
* Includes scaffolding for dog model

### CODE REVIEW

File: app/views/dogs/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<h1 class="dog-index-title">Dogs</h1>
+
+<a href="<%= dogs_url %>">
+<div class="dog-index-container">
```
Comment (epylinkn) [inline app/views/dogs/index.html.erb:4]: Can you fix this indentation?

File: app/views/dogs/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      <%= image_tag 'nana.png' %>
+      <p class="dog-show-link"><%= link_to dog.name, dog %></p>
+    </div>
+  <% end %>
+</div>
+</a>
+<br>
+<div class="new-dog">
+<%= link_to 'New Dog', new_dog_path, class: "new-dog-link" %>
+</div>
```
Comment (epylinkn) [inline app/views/dogs/index.html.erb:16]: Github doesn't like it when you don't have a new line at end of file. Can you configure VSCode to do it?

File: config/routes.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  resources :dogs
+  get 'hello_world', to: 'hello_world#index'
```
Comment (epylinkn) [inline config/routes.rb:3]: Let's get rid of this too.

**Review by epylinkn — CHANGES_REQUESTED** [review]
Looks good! Some code that looks like it should be in here.

File: app/views/dogs/index.html.erb (hunk 1 above)
Comment (cat-does) [reply app/views/dogs/index.html.erb:4]: Removed outer link, created a story to add it back.

**Pull request merged**